Vagrantfile

```diff
@@ -48,7 +50,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |c|
     end
 
     if $mount_users_dir
-      config.vm.synced_folder $home, $home, id: "core", :nfs => true, :mount_options => ['nolock,vers=3,udp']
+      config.vm.synced_folder $home, $home, type: "nfs"
```

Comment:
I think you guys are being a little paranoid - we generally are using well-known images and they're running locally. By default docker-machine also exposes all of $HOME - it just makes thing simpler when you're trying to volume mounts into Docker containers.